growth/too/tasks/growthdb_cgi.py

```diff
@@ -19,14 +21,87 @@
 """
 
 
+def prepare_candidates_for_object_table(sources_all):
+    """Prepare a list of disctionaries with relevant info
```

Comment:
I can't tell from this function docstring what the arguments and output are, or what its purpose is. Would you please add a numpydoc-style docstring to explain what it does and what its argument and return types are?

Reply:
Done.